samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -0,0 +1,297 @@
+
+.. _nrf_cloud_mqtt_multi_services_sample_requirements:
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :sample-yaml-rows:
+
+.. _nrf_cloud_mqtt_multi_services_sample_features:
+
+Features
+********
+
+This sample implements or demonstrates the following features:
+
+* Error-tolerant use of the `nRF Cloud MQTT API`_ using the :ref:`nrf_modem <nrfxlib:nrf_modem>` and :ref:`nrf_cloud <lib_nrf_cloud>` libraries.
+* Support for `Firmware Over The Air (FOTA) <nRF Cloud Getting Started FOTA documentation_>`_ via the `nRF Cloud`_ portal.
+* Support for `modem AT Commands <AT Commands Reference Guide_>`_ over UART using the :ref:`AT Host library<lib_at_host>`.
+* Periodic cellular and GNSS location tracking using the :ref:`location library <lib_location>`.
+* Periodic temperature sensor sampling on your `Nordic Thingy:91`_, or fake temperature  measurements on your `Nordic nRF9160 DK`_.
+* Transmission of sensor and GNSS location samples to the `nRF Cloud`_ portal as `nRF Cloud Device Messages`_.
+* Construction of valid `nRF Cloud Device Messages_` using `cJSON`_.
+
+
+.. _nrf_cloud_mqtt_multi_services_sample_structure:
+.. _nrf_cloud_mqtt_multi_services_sample_structure_and_theory:
+.. _nrf_cloud_mqtt_multi_services_sample_structure_and_theory_of_operation:
+.. _nrf_cloud_mqtt_multi_services_sample_theory_of_operation:
+
+Structure and Theory of Operation
+*********************************
+
+This sample is separated into a number of smaller functional units.
+The top level functional unit and entry point for the sample is :file:`src/main.c`.
+This file starts three threads, each with a distinct function:
+
+* Connection Thread (``conn_thread``): Runs the `Connection Management Loop <nrf_cloud_mqtt_multi_services_sample_connection_management_loop_>`_, which maintains a connection to the `nRF Cloud MQTT API`_.
+* Application Thread (``app_thread``): Runs the `Main Application Loop <nrf_cloud_mqtt_multi_services_sample_main_application_loop_>`_, which may add `Device Messages <nRF Cloud Device Messages_>`_ to the `Device Message Queue <nrf_cloud_mqtt_multi_services_sample_device_message_queue_>`_.
+* Message Thread (``msg_thread``): Processes the `Device Message Queue <nrf_cloud_mqtt_multi_services_sample_device_message_queue_>`_ whenever there is an active connection.
+
+.. _nrf_cloud_mqtt_multi_services_sample_connection_management_loop:
+
+Connection Management Loop
+==========================
+
+The Connection Management Loop performs four primary tasks:
+
+1) Configure and activate the cellular modem.
+#) Start a connection to the LTE Radio Access Network (RAN).
+#) Maintain a connection to the `nRF Cloud MQTT API`_.
+#) Reestablish or retry all lost or failed connections.
+
+Modem setup and the RAN connection are set-and-forget.
```

Comment:
Is "RAN" used elsewhere in the NCS docs?

Reply:
No, definitely not, you're right. I think other Nordic documentation just refers to it as "LTE". Maybe that is the better choice?

I went for RAN since it is the technology-agnostic term -- For instance, LTE/4G's RAN is, specifically, "EUTRAN": Evolved UMTS Terrestrial Radio Access Network". 3G's RAN was UTRAN: "UTMS Terrestrial Radio Access Network". 

I'm really crossed between the two terms. RAN feels like the "right" term, to me, but it is admittedly very rarely used. LTE is certainly the more commonly recognized term. Do you think I should switch to that?

Reply:
I would follow precedent in other docs.  LTE is more common.  Leave it up to the docs team to make site-wide terminology changes when growth in Nordic's product offerings justifies it.  For example, we used to use GPS exclusively, and then eventually changed it to GNSS because the nRF9160 modem firmware now supports QZSS, and in the future, this or other products may support Galileo or other constellations.

Reply:
That makes sense, thank you